### ISSUE TITLE
[Request] PSP Adaptation

### ISSUE THREAD

**Issue by TheMrIron2** [LFalch/hnefatafl#20]
Hi,

This isn't a formal request, but I would like to adapt some of this code to work on the PSP and may need a bit of help doing so. [Rust-PSP](https://github.com/overdrivenpotato/rust-psp) is currently a work-in-progress (std support, primarily, is the main culprit stopping Tafl from compiling right now) but I think it would be a good way to get this project some coverage, and a good way for some of the PSP guys to make a demo with Rust.
I have a [discord server for PSP programming](https://discord.gg/bePrj9W) with about 2,500 people, and a lot of Rust guys there would be delighted to see something happen. The JS code would need to be replaced - possibly with a substitute for the PSP's networking or local multiplayer - but the Rust core could largely be reused, I would imagine.

As I said, right now I can't do much until Rust-PSP reaches a more complete stage in the coming weeks, but I'd like to know if you guys would be interested in a fun project like this. I am not expecting much but I thought it wouldn't hurt to ask :)

**Comment by LFalch** [comment]
Yeah, that sounds like an amazing idea! I'll join the Discord and come say hi :)
